### ISSUE TITLE
🐝 [Bug] The Demon Scarred Cloak

### ISSUE THREAD

**Issue by ghost** [vmangos/core#2300]
## 🐝 Bug report - Quest
On the latest build, the quest 770 (https://www.wowhead.com/classic/quest=770/the-demon-scarred-cloak) can be repeated.
The item that starts the quest (https://www.wowhead.com/classic/item=4854/demon-scarred-cloak) does not disappear after completion, which is perfectly normal, but the quest can be repeated right after, which is wrong.

### Expected behavior
This quest should be doable only once.

Client Version:
1.12.1.5875

**Comment by Gamemechanicwow** [comment]
are you able to accept the quest a 2nd time?

From what I can tell its blizzlike that:

- the item stays in the bag.
- it popup with the quest when rightclicked
- you can't accept it again, once completed.

**Comment by ghost** [comment]
I didn't try to re-accept it to be honest as I do a legit playthrough, but I tried to right click on it after completing it and the quest popup opened with the description, the objective and the choice to do it or not (like when I clicked on it the first time), after some searching It seems the popup shouldn't open at all after the quest has been completed, this comment https://www.wowhead.com/classic/quest=770/the-demon-scarred-cloak#comments:id=2780079:reply=1327590 said he had an error message, which I didn't had.

I'm not 100% sure though, maybe it's intended, maybe not, sorry if it is.

**Comment by Gamemechanicwow** [comment]
Maybe it is a minor bug,

I would rephrase it to: Should the errormessage be displayed when rightclicking the item - or after trying to re-accept the quest when trying to start a quest that you have already accepted,

> I'm not 100% sure though, maybe it's intended, maybe not, sorry if it is.

Don't let me stand in the way of your bugreports or in anyway disscourage you. We are all here for the same thing :>

**Comment by FlagFlayer** [comment]
I've done this quest on Classic before multiple times. You get the "You are already on that quest" error when right clicking the cloak, the quest window does not open. Although, it's worth stating that just because this is how Classic handles it that doesn't necessarily mean that's how it was handled in vanilla.
